### PR TITLE
feat: Unify padding on all pages

### LINKED ISSUE #1108: Unify padding on all pages.
**Is your feature request related to a problem? Please describe.**
Most pages have the same padding as the summary page, but the `Data & Storage` and `Statistics` pages have more padding as shown in the picture below. In my personal opinion, wide padding like `Data & Storage` pages seems less stuffy, but it can reduce the amount of data that can be displayed at one time, so you have to decide which one is better.
[Summary page]
image
[Data & Storage page]
image
[Statistics page]
image

**Describe the solution you'd like**
Unify padding on all pages.

### PR DESCRIPTION
**Pull request by yujung7768903**
resolve #1108 

* set padding of div element(id="app-page") to 14px
스크린샷 2021-08-31 오전 2 54 00

* set margin of div element(class="card") to 14px
스크린샷 2021-08-31 오전 2 59 41

* conclusion
I thought it would be better if the gap between content and the gap between content and container were the same size.
So I set padding and margin to 14px.
스크린샷 2021-08-31 오전 3 22 50

### CODE REVIEW

File: src/components/backend-ai-session-view.ts [hunk 1]
```diff
@@ -431,15 +431,13 @@ export default class BackendAiSessionView extends BackendAIPage {
   render() {
     // language=HTML
     return html`
-      <div class="horizontal layout wrap">
```
Comment (agatha197) [inline src/components/backend-ai-session-view.ts:434]: If you remove this line, the resource statistics panel is too wide. Why don't you remain this line?

Comment (yujung7768903) [reply src/components/backend-ai-session-view.ts:434]: This change is related to lablup/backend.ai-webui/pull/1117, lablup/backend.ai-webui/issues/1113.
I removed this line to make the width of resource statics panel and the width of session list panel equal.

Comment (agatha197) [reply src/components/backend-ai-session-view.ts:434]: I got it. LGTM!

**Review by inureyes — APPROVED** [review]
LGTM

**Pull request merged**